WcaOnRails/app/models/competition.rb

```diff
@@ -640,7 +640,11 @@ def in_progress?
   end
 
   def is_over?
-    !end_date.nil? && end_date < Date.today
+    # We treat "today == last day of competition" as the competition is over.
+    # Once we know the schedules and timezones of competitions, we'll be able
+    # to do something more accurate here.
+    # See https://github.com/thewca/worldcubeassociation.org/issues/737
+    !end_date.nil? && end_date <= Date.today
```

Comment:
This seems wrong.

Isn't the model is now incorrect, since `is_over?` returns true before the competition is actually over? Even if that doesn't cause any problems in the other 8 locations calling the method, that's not good.

Changing when posting results is allowed, should probably be done in the posting code. Or this method could be changed to work like this: `competition.is_over?(in: 1.day.from_now)`.